spec/v3/services/repository/find_spec.rb

```diff
@@ -123,6 +123,35 @@
     include_examples '404 not found'
   end
 
+  describe "fetching a case sensitive repository by slug Minimal when only minimal is defined" do
```

Comment:
I think this example is misleading.

Reply:
You are right @BanzaiMan.
Not needed after implementation was changed.